src/Misc/layoutbin/update.sh.template

```diff
@@ -135,7 +135,7 @@ if [[ "$currentplatform" == 'darwin'  && restartinteractiverunner -eq 0 ]]; then
     then
         # inspect the open file handles to find the node process
         # we can't actually inspect the process using ps because it uses relative paths and doesn't follow symlinks
-        path=$(lsof -a -g "$procgroup" -F n | grep node12/bin/node | grep externals | tail -1 | cut -c2-)
+        path=$(lsof -a -g "$procgroup" -F n | grep node16/bin/node | grep externals | tail -1 | cut -c2-)
```

Comment:
should the node16 changes wait on `fhammerl`'s pr?

Reply:
Yes, i will rebase on his change once it's merged.

Reply:
Depends on #1621

Reply:
Looks like #1621 has been merged.  Can this PR now be merged?  Thanks.  This is important to us.

Reply:
We're still working on the dependencies for this - here's some more information. https://github.com/actions/runner/issues/805#issuecomment-1041968059

Reply:
I can see a lot of activity in Github org about nodejs 16 which is exciting!

@ethomson Is that list still up to date?